docs/_integration-with-existing-apps-ios.md

```diff
@@ -0,0 +1,596 @@
+}
+
+ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
++ self.automaticallyLoadReactNativeWindow = NO;
++ return [super application:application didFinishLaunchingWithOptions:launchOptions];
+   window = [UIWindow new];
+   window.rootViewController = [ViewController new];
+   [window makeKeyAndVisible];
+   return YES;
+
+ }
+
++- (NSURL *)sourceURLForBridge:(RCTBridge *)bridge
++{
++  return [self bundleURL];
++}
+
++- (NSURL *)bundleURL
++{
++#if DEBUG
++  return [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"index"];
++#else
++  return [[NSBundle mainBundle] URLForResource:@"main" withExtension:@"jsbundle"];
++#endif
++}
+ @end
+```
+
+Let's have a look at the code above:
+
+1. We are inherithing from the `RCTAppDelegate` and we are calling the `applicatinon:didFinishLaunchingWithOptions` of the `RCTAppDelegate`. This delegates all the React Native initialization processes to the base class.
+2. We are customizing the RCTAppDelegate by setting the `automaticallyLoadReactNativeWindow` to `NO`. This step instruct React Native that the app is handling the `UIWindow` and React Native should not worry about that.
+3. The methods `sourceURLForBridge:` and `bundleURL` are used by the App to tell to React Native wher it can find the JS bundle that needs to be rendered. The `sourceURLForBridge:` is from the Old Architecture and you can see that it is deferring the decision to the `bundleURL` method, required by the New Architecture.
+
+</TabItem>
+<TabItem value="swift">
+
+To achieve this, we have to modify the `AppDelegate.swift`
+
+1. Open the `AppDelegate.swift` files and modify it as it follows (See the official template's [AppDelegate.swift](https://github.com/react-native-community/template/blob/main/template/ios/HelloWorld/AppDelegate.swift) as reference):
+
+```diff title="AppDelegate.swift"
+import UIKit
++import React_RCTAppDelegate
++import React_RCTAppDelegate
+
+@main
+-class AppDelegate: UIResponder, UIApplicationDelegate {
++class AppDelegate: RCTAppDelegate {
+
+-  var window: UIWindow?
+
+-  func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
++  override func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    // Override point for customization after application launch.
++    self.automaticallyLoadReactNativeWindow = false
++    super.application(application, didFinishLaunchingWithOptions: launchOptions)
+    window = UIWindow()
+-    window?.rootViewController = ViewController()
+-    window?.makeKeyAndVisible()
++    window.rootViewController = ViewController()
++    window.makeKeyAndVisible()
+    return true
+  }
+
++  override func sourceURL(for bridge: RCTBridge) -> URL? {
++    self.bundleURL()
++  }
+
++  override func bundleURL() -> URL? {
++#if DEBUG
```

Comment:
It might be worth mentioning that users should check Active Compilation Conditions. In theory this should be set but if its not then this may lead to some issues 

![image](https://github.com/user-attachments/assets/ef0bc957-b014-4a26-a06e-3c43e24a0a70)

Reply:
Debug and Release flavors are created by default by Xcode. The only case when that flag is not there is for a project that started as Objective-C and got migrated to Swift - so Xcode didn't add that automation. 
But that's an edge case that people working on migrated projects should have already handled.
I'd rather keep the guide minimal, if possible.